app/Console/Commands/FetchCollectionBanner.php

```diff
@@ -17,29 +16,29 @@ class FetchCollectionBanner extends Command
      *
      * @var string
      */
-    protected $signature = 'nfts:fetch-collection-banner {--collection-id=} {--missing-only}';
+    protected $signature = 'nfts:fetch-collection-banner {--collection-id=}';
 
     /**
      * The console command description.
      *
      * @var string
      */
-    protected $description = 'Fetch the latest banner for collections';
+    protected $description = 'Fetch the latest banner for a collection';
 
     /**
      * Execute the console command.
      */
     public function handle(): int
     {
-        $queryCallback = function (Builder $query): Builder {
-            return $this->option('missing-only')
-                        ? $query->whereNull('extra_attributes->banner')
-                        : $query;
-        };
+        if (empty($this->option('collection-id'))) {
+            $this->error('The `collection-id` is missing. Please either set `collection-id` flag or use `nfts:fetch-collection-banner-batch` command to fetch banner of multiple collections');
+
+            return Command::INVALID;
+        }
```

Comment:
Enforced `collection-id` flag to ensure that we are using the correct job to fetch banners of multiple collections.